src/app/dts/dts.component.html

```diff
@@ -11,6 +11,7 @@
   [headerHeight]="50"
   [footerHeight]="50"
   [rowHeight]="50"
+  (activate)='rowClick($event)'
```

Comment:
I'm being nitpicky, but this attribute is the only one with single quotes. All of the other ones use double quotes.

Reply:
Fixed.